src/main/java/at/hannibal2/skyhanni/config/features/chat/StashConfig.kt

```diff
@@ -23,17 +24,33 @@ class StashConfig {
     var notice: String = ""
 
     @Expose
-    @ConfigOption(name = "Hide Added Messages", desc = "Hide the messages when something is added to your stash.")
-    @ConfigEditorBoolean
-    var hideAddedMessages: Boolean = true
+    @ConfigOption(name = "Hide Dupe Warnings", desc = "")
+    @Accordion
+    var hideDuplicateWarning: HideDuplicateWarningConfig = HideDuplicateWarningConfig()
+
+    class HideDuplicateWarningConfig {
+        @Expose
+        @ConfigOption(
+            name = "Hide Dupe Warnings",
+            desc = "Hide duplicate warnings for previously reported stash counts."
+        )
+        @ConfigEditorBoolean
+        var enabled: Boolean = true
+
+        @Expose
+        @ConfigOption(
+            name = "World Change Reset",
```

Comment:
the term reset seems weird here. how about "ignore on world switch" and inverting this check?

Reply:
i understand that the term reset matches the technical logic, but its not intuitive to show this term in the config.

Reply:
See if you like the new wording better